### ISSUE TITLE
`hlx demo` & `hlx up` fails

### ISSUE THREAD

**Issue by stefan-guggisberg** [adobe/helix-cli#1189]
**Description**
Starting the local development server with `hlx up` in a new project created with `hlx demo` fails with the following error:
```
Unable to determine origin url for --local-repo = .
```

**To Reproduce**
1. Open terminal and enter
```
$ hlx demo demo
$ cd demo
$ hlx up
Unable to determine origin url for --local-repo = .
```

**Expected behavior**
`hlx demo` follow by `hlx up` should work out of the box

**Version:**
```bash
$ hlx --version
5.7.0
```

**Additional context**
As a workaround you can use `hlx up --no-local-repo` instead of `hlx up`

**Comment by stefan-guggisberg** [comment]
regression of #913

**Comment by stefan-guggisberg** [comment]
closed via #1201
